modules/terraform-aws-ca-lambda/utils/certs/s3.py

```diff
@@ -39,3 +40,58 @@ def s3_upload(
         return s3_upload_file(file, external_s3_bucket_name, key, content_type)
 
     return s3_upload_file(file, internal_s3_bucket_name, key, content_type)
+
+
+def convert_to_json(input_str):
```

Comment:
do we want to rename this to `convert_x509_subject_str_to_dict` or something less ambiguous? it's not clear what it's converting and it's also not converting it to `json`

Reply:
thanks, changed as suggested